src/main/cpp/mechanisms/noteManager/decoratormods/readyOdometryLaunchState.cpp

```diff
@@ -13,60 +14,60 @@
 // This file was automatically generated by the Team 302 code generator version 1.2.1.0
-// Generated on Monday, January 29, 2024 6:28:21 AM
+// Generated on Friday, February 2, 2024 7:00:51 PM
 
 // C++ Includes
 #include <string>
 
 // FRC includes
 
 // Team 302 includes
-#include "mechanisms/Thing1Mech/generated/Thing1Mech_thing1Talon_StateGen.h"
-#include "mechanisms/Thing1Mech/decoratormods/Thing1Mech_thing1Talon_State.h"
+#include "mechanisms/noteManager/decoratormods/readyOdometryLaunchState.h"
 #include "teleopcontrol/TeleopControl.h"
 #include "teleopcontrol/TeleopControlFunctions.h"
 #include "utils/logging/Logger.h"
 
 // Third Party Includes
 
 using namespace std;
+using namespace noteManagerStates;
 
 /// @class ExampleForwardState
 /// @brief information about the control (open loop, closed loop position, closed loop velocity, etc.) for a mechanism state
-Thing1Mechthing1TalonState::Thing1Mechthing1TalonState(std::string stateName,
-													   int stateId,
-													   Thing1Mechthing1TalonStateGen *generatedState,
-													   Thing1Mech *mech) : State(stateName, stateId), m_genState(generatedState), m_mechanism(mech)
+readyOdometryLaunchState::readyOdometryLaunchState(std::string stateName,
+												   int stateId,
+												   noteManagerAllStatesStateGen *generatedState,
+												   noteManager *mech) : State(stateName, stateId), m_genState(generatedState), m_mechanism(mech)
 {
 }
 
-void Thing1Mechthing1TalonState::Init()
+void readyOdometryLaunchState::Init()
 {
-	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("Thing1Mechthing1TalonState"), string("init"));
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("readyOdometryLaunchState"), string("init"));
 
 	m_genState->Init();
 }
 
-void Thing1Mechthing1TalonState::Run()
+void readyOdometryLaunchState::Run()
 {
-	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("Thing1Mechthing1TalonState"), string("run"));
+	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("readyOdometryLaunchState"), string("run"));
 	m_genState->Run();
 }
 
-void Thing1Mechthing1TalonState::Exit()
+void readyOdometryLaunchState::Exit()
 {
 	m_genState->Exit();
 }
 
-bool Thing1Mechthing1TalonState::AtTarget()
+bool readyOdometryLaunchState::AtTarget()
 {
 	auto attarget = m_genState->AtTarget();
 	return attarget;
 }
 
-bool Thing1Mechthing1TalonState::IsTransitionCondition(bool considerGamepadTransitions)
+bool readyOdometryLaunchState::IsTransitionCondition(bool considerGamepadTransitions)
 {
 	// To get the current state use m_mechanism->GetCurrentState()
 
-	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXPEL));
+	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXAMPLE_MECH_FORWARD));
```

Comment:
Why is this using an ExampleMech teleop control function?

Reply:
Base generated code as an example not mapped to anything, this won't get updated until Auton implements things

Reply:
Ok